### PR TITLE
Moved GitBook documentation to GitHub

### PR DESCRIPTION
**Pull request by cwarnermm**

### CODE REVIEW

File: README.md
Comment (mickmister) [inline README.md:90]: The diff is `+161 -266`, which I would expect to be generally equal. It seems there is no content from the deleted files put into the README. Also, everything from line 88 and down is related to `build/sync` folder, and not the Nagios plugin.

@cwarnermm What's the intended outcome of this PR? I assume it's eant to move content from `docs` into the README but that doesn't seem to be what's going on here

Comment (cwarnermm) [reply README.md:90]: Wow, did these commits get mangled up! I experienced a strange error mid-way through this work and suspect that what we're seeing in the commits is related.

I'll review and correct where needed, then let you know when this PR is ready.

**Comment by cwarnermm** [comment]
@mickmister - This PR should now be ready for your review.

File: README.md
Comment (mickmister) [reply README.md:308]: It looks like the sync stuff is still here. This is unrelated to this plugin, and should probably be removed from the README

File: README.md
Comment (mickmister) [inline README.md:245]: I'm not sure if this section should be here either

Comment (cwarnermm) [reply README.md:245]: If it's no longer applicable to the plugin, let's remove it.

File: README.md
Comment (cwarnermm) [reply README.md:308]: Yes, the Sync content was migrated over. If it's no longer relevant to the plugin, we can remove it.

Comment (mickmister) [reply README.md:308]: So the sync content is from https://github.com/mattermost/mattermost-plugin-nagios/blob/master/build/sync/README.md. The `build` folder (and as a result the `build/sync` folder) is essentially duplicated from the starter template, and this sync folder was actually recently removed from the template since it's never used. We should avoid migrating the sync folder README into the plugin repo READMEs

https://github.com/mattermost/mattermost-plugin-starter-template/pull/185#discussion_r1269908082
https://github.com/mattermost/mattermost-plugin-starter-template/pull/186

Comment (mickmister) [reply README.md:308]: Sorry for the confusion on this. The sync tool seemed like a good idea, but never took off, and it's sort of been taking up space in each plugin repo since then. Really, we should remove it from all plugin repos.

Comment (cwarnermm) [reply README.md:308]: Agreed 100%. All of the plugin docs hosted via GitBooks has had GitHub sync'ing disabled, thanks to @emdecr. Now that sync is disabled, we're simply cleaning up/removing those ``/docs`` folders where all content gets moved to the README.

I really appreciate your double-checks to ensure that what I'm migrating over should be migrated, @mickmister!

File: README.md
Comment (mickmister) [reply README.md:245]: Where did this `Nagios Core API Client` content come from? I don't see it in the removed diff

> Is there someone I should add as a reviewer on this PR to identify all content that can be safely omitted from the README page?

The community members that developed the plugin may be able to provide some insight @amwolff & @DanielSz50. @DHaussermann may also have some insight on this

Comment (cwarnermm) [reply README.md:245]: The ``Nagios Core API Client`` content was published via GitBook.

File: README.md
Comment (mickmister) [reply README.md:308]: > All of the plugin docs hosted via GitBooks has had GitHub sync'ing disabled,

I'd like to clarify that the `build/sync` tool is not the same GitBooks syncing thing. Not sure if we had accidentally crossed some shared terminology here

Comment (cwarnermm) [reply README.md:308]: I did misinterpret the context here! Thank you for the confirmation about being able to remove the sync content from the README. I'll do so via a commit on this PR (along with the other content removal pending confirmation)

**Review by mickmister — APPROVED** [review]
LGTM, assuming deletion of `## Nagios Core API Client` and `## Sync` sections before merge

**Comment by cwarnermm** [comment]
@mickmister - This PR appears stuck at the check stage. Is there a way to get it moving again?

**Pull request merged**
